docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+---
+id: data-types
+title: Data Types Overview
+sidebar_label: Data Types and File Formats
+---
+
+ICGC ARGO provides access to harmonized data from new programs
```

Comment:
add a period.

Also, did Christina still want to stay away from the word "harmonized"?

Reply:
did you remember what word to use instead?

Reply:
changed to: ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types).

Reply:
yeah that sounds good...
I think it was just using "analyzed" instead...I'd have to look in my notes